### PR TITLE
DP-24553 upgrade python from v2 to v3

### PR DESCRIPTION
**Pull request by clairesunstudio**
To test:
- docker pull `circleci/python:3.10`
- ddev service enable backstop
- Test backstop commands in circleCI --> `ddev drush ma:backstop tugboat prod --tugboat=https://pr1499-fl4c3dl1elgsafswerlmux9aqikrynsi.tugboat.qa/ --ci-branch=DP-24553/upgrade-circleci-python`

`ddev drush ma:backstop cd prod --ci-branch=DP-24553/upgrade-circleci-python`

Backstop run in CD https://app.circleci.com/pipelines/github/massgov/openmass/19062/workflows/4af91a45-756e-4730-a2ee-76d268c19df3/jobs/79123/artifacts

----

Other backstop commands: https://github.com/massgov/openmass/blob/develop/backstop/README.md

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -427,7 +427,7 @@ jobs:
     resource_class: xlarge
     docker:
       # Need to use docker image for these steps.
-      - image: circleci/python:2.7.14
+      - image: circleci/python:3.10
```
Comment (clairesunstudio) [inline .circleci/config.yml:430]: why are we using python image here? Mayflower is using `image: cimg/node:12.14`. Should we make them consistent?

Comment (clairesunstudio) [reply .circleci/config.yml:430]: Although, the updated python image seems to work in circleCI for openmass, see report https://app.circleci.com/pipelines/github/massgov/openmass/19062/workflows/4af91a45-756e-4730-a2ee-76d268c19df3/jobs/79123/artifacts, when I pull down the image and run patternlab backstop locally its having issues displaying the images. 

Screen Shot 2022-05-09 at 8 40 23 PM

Comment (clairesunstudio) [reply .circleci/config.yml:430]: nvm, this could be totally unrelated...

**Comment by deviantintegral** [comment]
We switched to using `- image: cimg/node:16.15` so we could run jobs on the faster CI servers. This work is no longer needed 🙌 .

**Pull request closed without merging**